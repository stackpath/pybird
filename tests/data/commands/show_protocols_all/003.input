
0001 BIRD 2.0.4 ready.
2002-Name       Proto      Table      State  Since         Info

1002-foo_igp     OSPF       foo_net up     2019-05-31    Running
1006-  Description:    Foo IGP
   Channel ipv4
     State:          UP
     Table:          foo_network
     Preference:     150
     Input filter:   (unnamed)
     Output filter:  REJECT
     Import limit:   1000
       Action:       disable
     Routes:         3 imported, 0 exported, 1 preferred
     Route change stats:     received   rejected   filtered    ignored   accepted
       Import updates:              6          0          0          0          6
       Import withdraws:            3          0        ---          0          3
       Export updates:             47          4         43        ---          0
       Export withdraws:           22        ---        ---        ---          0

0000