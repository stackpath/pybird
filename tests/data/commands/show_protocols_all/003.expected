[
    {"export_updates_received": 47, 
    "protocol": "OSPF", 
    "description": "Foo IGP", 
    "ipv4_state": "UP", 
    "import_updates_ignored": 0, 
    "import_withdraws_rejected": 0, 
    "export_updates_accepted": 0, 
    "import_updates_accepted": 6, 
    "import_updates_received": 6, 
    "export_updates_filtered": 43, 
    "import_withdraws_ignored": 0, 
    "routes_imported": 3, 
    "export_updates_rejected": 4, 
    "import_updates_filtered": 0, 
    "export_withdraws_accepted": 0, 
    "import_withdraws_accepted": 3, 
    "export_withdraws_received": 22, 
    "up": false, 
    "name": "foo_igp", 
    "state": "Running", 
    "routes_exported": 0, 
    "import_updates_rejected": 0, 
    "last_change": "2019-05-31T00:00:00", 
    "import_withdraws_received": 3
    }
]